wildvvad/utils/helper_functions.py

```diff
@@ -0,0 +1,115 @@
+"""
+This callback is inspired by the CSVLogger callback from Keras.
+reference:
+    https://keras.io/api/callbacks/csv_logger/
+"""
+import collections
+import csv
+import time
+import numpy as np
+import tensorflow as tf
+keras = tf.keras
+
+
+def get_lr_metric(optimizer):
+    def lr(y_true, y_pred):
+        return optimizer.learning_rate
+    return lr
+
+class CSVLogger(keras.callbacks.Callback):
```

Comment:
how is this different to https://keras.io/api/callbacks/csv_logger/

Reply:
It can be adjusted to specific needs, I used the base implementation (minimum)